.github/workflows/release.yaml

```diff
@@ -13,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout httpagent repository
-        uses: 'actions/checkout@v3'
```

Comment:
you should create a `.github/dependabot.yml` file to let it do that

Reply:
I agree, but meanwhile it adds a lot of noise and today we almost never merge dependabot PRs (usually because CI fails, as dependabot is often missing permissions)...

Reply:
Let's try it: https://github.com/scality/httpagent/pull/10/commits/0e7e5d2a76eac72be8d4047c91f532d3f831c7a5

Reply:
@williamlardier here's a couple of things to note in case of need:

- If you have permissions issue when a PR is triggered by dependabot, explicitly set the [`permissions` in your workflow](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#defining-access-for-the-github_token-permissions), it should fix most of the issues if not all (contact me if not).
- when a repo is using git waterflow, always target the latest branch (the default branch if that's the case) as waterflow + dependabot does not work well together. So the PRs are going to be ignored indeed.
- Lastly you can now group dependabot PRs this reduce the number of prs to review/merge and can help make the job less tedious. (Ping me if you'd like to know more about it).